pennylane/qnn/torch.py

```diff
@@ -302,14 +308,14 @@ def forward(self, inputs):  # pylint: disable=arguments-differ
             tensor: output data
         """
 
-        if len(inputs.shape) > 1:
+        if self.split_batches and len(inputs.shape) > 1:
```

Comment:
Same as for keras regarding multiple batch dimensions.

Reply:
https://github.com/PennyLaneAI/pennylane/pull/4131#discussion_r1196678630